frontend/ui/data/koptoptions.lua

```diff
@@ -33,7 +34,7 @@ local KoptOptions = {
                 alternate = false,
                 values = {0, 1, 2},
                 default_value = DKOPTREADER_CONFIG_TRIM_PAGE,
-                enabled_func = function() return require("device"):isTouchDevice() end,
+                enabled_func = function() return Device:isTouchDevice() end,
```

Comment:
Is there a problem `enabled_func = Device.isTouchDevice,`?

Reply:
I did it like the widget above 
```                
                name = "screen_mode",
                name_text = S.SCREEN_MODE,
                toggle = {S.PORTRAIT, S.LANDSCAPE},
                alternate = false,
                args = {"portrait", "landscape"},
                default_arg = "portrait",
                current_func = function() return Screen:getScreenMode() end,
                event = "SetScreenMode",
            }
```

Plus it doesn't work, fail with 
```
./luajit: error loading module 'ui/data/koptoptions' from file 'frontend/ui/data/koptoptions.lua':
        frontend/ui/data/koptoptions.lua:37: function arguments expected near ','
```

Reply:
Could you paste the entirety of what you're trying that fails?